src/v2/server.py

```diff
@@ -30,6 +33,16 @@ def get_player(self, invocation, app_id, unused_options):
             self.target_object.registry.add_player(player)
         invocation.return_value(GLib.Variant("(o)", (player.object_path, )))
 
+    def _player_added_cb(self, unused_registry, player):
+        self.server.watch_bus_name(player.bus_name)
+        self.server.cancel_countdown()
+        self.server.hold()
```

Comment:
This commit keeps the server alive until the last player gets removed, which happens when the last application gets closed/killed. I think now it goes against the autoquit feature. So probably this commit shouldn't be accepted.

Reply:
I think the server should be alive while any Sounds are still active, but active Players should not prevent the server from quitting. That would most closely match the old (and correct IMO) behaviour.

Reply:
Then this commit can be ignored. You could review the other commits.